### PR TITLE
Update setup-anti-phishing.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated anti phising settings

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5914

File: microsoft-365/business-video/setup-anti-phishing.md [hunk 1]
```diff
@@ -31,16 +31,13 @@ Phishing is a malicious attack where an email looks like it was sent from a fami
...
 1. Select **Default Policy** to refine it.
-1. In the **Impersonation** section, select **Edit**.
-1. Go to **Add domains to protect** and select the toggle to automatically include the domains you own.
-1. Go to **Actions**, open the drop-down **If email is sent by an impersonated user**, and choose the action you want.
-
-    Open the drop-down **If email is sent by an impersonated domain** and choose the action you want.
-1. Select **Turn on impersonation safety tips**. Choose whether tips should be provided to users when the system detects impersonated users, domains, or unusual characters. Select **Save**.
-1. Select **Mailbox intelligence** and verify that it's turned on. This allows your email to be more efficient by learning usage patterns.
+1. In the **Phishing threshold & protection** section, select **Edit protection settings**.
+1. Choose **Enable domains to protect** and select **Include the domains you own**, **Include custom domain** and Click on **Manage custom domain(s)** to add a domain.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/setup-anti-phishing.md]:
```suggestion
1. Choose **Enable domains to protect** and select **Include the domains you own** and **Include custom domain**, then select **Manage custom domain(s)** to add a domain.
```

File: microsoft-365/business-video/setup-anti-phishing.md [hunk 2]
```diff
@@ -31,16 +31,13 @@ Phishing is a malicious attack where an email looks like it was sent from a fami
...
 1. Choose **Add trusted senders and domains**. Here you can add email addresses or domains that shouldn't be classified as an impersonation.
-1. Choose **Review your settings**, make sure everything is correct, select **Save**, then **Close**.
+1. Choose **Enable Mailbox intelligence**, and **Enable Intelligence for impersonation protection** to enable enhanced impersonation results based on each user's individual sender map.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/setup-anti-phishing.md]:
```suggestion
1. Choose **Enable Mailbox intelligence** and **Enable Intelligence for impersonation protection** to enable enhanced impersonation results based on each user's individual sender map.
```

File: microsoft-365/business-video/setup-anti-phishing.md [hunk 3]
```diff
@@ -31,16 +31,13 @@ Phishing is a malicious attack where an email looks like it was sent from a fami
...
+1. Select **Enable Spoof intelligence** to Choose how you want to filter email from senders who are spoofing domains.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/setup-anti-phishing.md]:
```suggestion
1. Select **Enable Spoof intelligence** to choose how you want to filter email from senders who are spoofing domains.
```

File: microsoft-365/business-video/setup-anti-phishing.md [hunk 4]
```diff
@@ -31,16 +31,13 @@ Phishing is a malicious attack where an email looks like it was sent from a fami
...
+1. **Save**, then **Close**.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/setup-anti-phishing.md]:
```suggestion
1. Select **Save**, then **Close**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder.

**Pull request merged**